agora.md

```diff
@@ -0,0 +1,288 @@
+---
+layout: page
+title: AgoraDesk
+subtitle: Peer-to-peer cryptocurrency marketplace
+#cover-img: /assets/img/path.jpg
+share-img: https://bitcoiner.guide/assets/img/QNA_SHARE_IMG.png
+---
+<p align="center">
+  <a href="#what-is-agora-desk">Intro</a> |
+  <a href="#before-you-start">Preparation</a> |
+  <a href="#buying-bitcoin">Buy Sats</a> |
+  <a href="#faq">FAQ</a> |
+  <a href="#learn-more">Learn More</a> 
+</p>
+
+<p align="center">
+<img src="/assets/img/agoradesk.png" class=responsive style="max-width: 80%; height: auto;" />
+</p>
+
+## Introduction
+If you are new to Bitcoin, the most common way to acquire it is through a regulated exchange. However, the process involves providing a significant amount of personal information such as your name, address, banking details, and even a scan of your face or voice recording. You will also need to undergo background checks before being approved to make a purchase. 
+
+After all of that, you don't own any Bitcoin yet. You receive an IOU and still need to transfer the funds into a wallet where you control the private keys. This experience is not ideal and goes against the Ethos of Bitcoin. 
+
+Additionally, your personal information is now stored on the exchange's server, making it vulnerable to potential breaches. Unfortunately, such incidents are not uncommon in the Bitcoin world. Companies like 'Mt Gox', 'Binance', and 'Ledger' have all experienced data breaches in the past.
+
+## So what's the alternative?
+Buying Bitcoin [noKYConly!](/nokyconly) 
+
+There are many ways to obtain Bitcoin without providing all of your details to a company that may or may not know how to secure data properly. I have guides on using peer-to-peer markets like [Bisq](/bisq), [HodlHodl](/hodlhodl), [Robosats](/robosats) and [Peach Bitcoin](/peach) but another noKYC alternative is AgoraDesk.
+
+<p align="center">
+<img src="/assets/img/NKYC2.png" class=responsive width="300" height="230" maxheight="300" />
+</p>
+
+## What is AgoraDesk?
+
+AgoraDesk is an Open Source and online peer-to-peer marketplace for trading cryptocurrencies, especially designed to address the issue of KYC in Bitcoin. AgoraDesk was created by the same team that developed LocalMonero, which is an equivalent to LocalBitcoins for Monero. 
+
+With a vision to uphold the principles of privacy and freedom, AgoraDesk does not require any personal information while offering a wide range of payment methods with good liquidity on the most common. AgoraDesk is available on the clearnet, Tor network, and mobile applications on iOS and Android. 
+
+## Before You Start 
+
+### Things you need to know
+
+#### Jargon Buster
+
+|----|-------|
+| [Arbitration Bond](https://learn.robosats.com/docs/robots/) | Arbitration bond is a certain amount of Bitcoin that a seller locks in the platform as a guarantee of their credibility and seriousness about a trade. |
```

Comment:
All of these are linking to Robosats docs. Have you forgotten to change them?

Reply:
Nice catch, I completely missed this. I copied and pasted your RoboSats grid 😆